app/policies/application_policy.rb

```diff
@@ -74,4 +76,10 @@ def check_self_and_ancestors_for_role(role:, division: @record.division)
 
     false
   end
+
+  # Require admin role on at least one division to allow access to index view for divisions or users
+  def any_division_admin?
+    user.roles.where(name: 'admin', resource_type: 'Division').present?
```

Comment:
`.any?` is technically more efficient here since it will run a COUNT query instead of loading all the role objects and then checking for `empty?`. (`present?` == `!(nil? || empty?)`)